src/movies.js

```diff
@@ -1,16 +1,115 @@
 // Iteration 1: All directors? - Get the array of all directors.
+
+function getAllDirectors(movies) {
+    let alldirector = movies.map((movies) => movies.director);
+    return alldirector;
+  }
+
+
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors. How could you "clean" a bit this array and make it unified (without duplicates)?
 
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct?
 
+function howManyMovies(movies) {
+
+    let moviesd = movies.filter((movies) => movies.director === "Steven Spielberg" && movies.genre.includes("Drama"));
+    let thedramamovies = moviesd.length;
```

Comment:
As best practices, remember to make use of camel casing when naming your variables.